src/view/MovieListView.js

```diff
@@ -0,0 +1,65 @@
+import { Modal } from '../components/modal';
+import getMovieList from '../apis/getMovieList';
+import getSearchedMovies from '../apis/getSearchEdMovies';
+import { Cinema } from './Cinema';
+
+export function MovieListView() {
+    const cinema = new Cinema();
+    const moreButton = document.getElementById('more-movies');
+    const searchInput = document.getElementById('search-input');
+    const searchButton = document.querySelector('.search-button');
+
+    async function renderMovieItems(pages) {
+        try {
+            cinema.showSekeleton(20, true);
+            const itemList = await getMovieList(pages);
+            cinema.createMovies(itemList.results);
+        } catch (error) {
+            const modal = new Modal();
+            modal.content = error.message;
+            document.querySelector('body').appendChild(modal.rendered);
```

Comment:
```suggestion
new Modal(document.querySelector('body'), error.message);
```

이런식으로 표현할 수 있으면 어떨까 싶네요!

Reply:
훨씬더 가독성이 좋아진거같습니다...ㅠㅠ 더 간단한게 표현할 수 있는 방법을 생각하도록 하겠습니다!